src/DepthImageToLaserScanROS.cpp

```diff
@@ -47,36 +47,29 @@
 using namespace depthimage_to_laserscan;
 
 DepthImageToLaserScanROS::DepthImageToLaserScanROS(rclcpp::Node::SharedPtr & node):node_(node) {
-
-  cam_info_sub_ = node_->create_subscription<sensor_msgs::msg::CameraInfo>("depth_camera_info",
+  auto qos = rclcpp::QoS(rclcpp::KeepLast(10));
```

Comment:
We may as well just get rid of this and use `rclcpp::SensorDataQoS()`, basically everywhere below.